source/contracts/reporting/ReportingWindow.sol

```diff
@@ -84,6 +84,7 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
         require(_originalUniverse.isContainerForReportingWindow(_shadyReportingWindow));
         IReportingWindow _originalReportingWindow = _shadyReportingWindow;
         require(_originalReportingWindow.isContainerForMarket(_market));
+        _shadyReportingWindow.migrateFeesDueToFork();
```

Comment:
Recommend an intermediate variable `IReportingWindow legitReportingWindow = _shadyReportingWindow` on the line after the reporting window is deemed to be verified.

Also, just noticed this code doesn't call `_market` `_shadyMarket` like most of the other functions with this pattern.  Any unvalidated address should be named `_shady*` until it is validated, then renamed (copied) to `_legit*` (or no-prefix) to make it clear when validation has finished.